ansible/configs/base-infra/README.adoc

```diff
@@ -0,0 +1,169 @@
+== Overview
+
+*base-aap2-infra* is a Base Architecture building an Ansible Automation Platform on traditional Cloud VMs or Instances.  AWS is the first supported platform. Original first CI will be Automate the Hybrid Cloud demo (2023-02) where it will provide the base underlying infrastructure. 
+
+=== Cloud Support
+
+* AWS via `cloud_provider: ec2`
+* Adding other clouds *should* be relatively trivial as this is _just a bunch of instances_ at its core
+
+It is designed for the purpose of building a highly reliable and repeatable for structure for use with Ansible Automation Platform 2 (AAP2) demos and workshops.
```

Comment:
It think this whole readme needs updating

Reply:
It did, this config was derived from base-aap2-infra, updating readme


> There is a lot of extra "stuff" for a "base" config.

Valid point, cleaned and simpler now